src/Drivers/SlackNotificationDriver.php

```diff
@@ -16,14 +16,6 @@ public static function getSettingsView(): string
         return 'seatnotifications::slack.settings';
     }
 
-    /**
-     * @return string
-     */
-    public static function getRegistrationView(): string
-    {
-        return 'seatnotifications::slack.registration';
```

Comment:
if you do this the registration view will be empty. as this results in the driver configuration view 
![image](https://user-images.githubusercontent.com/6583519/53694942-7acad980-3db5-11e9-9e7a-5fded27473c4.png)